Final_Project/Project_Proposal.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "GEO 511 Project Proposal"
+author: "Collin O'Connor"
+date: "10/14/2020"
+output: github_document
+---
+
+<style type="text/css">
```

Comment:
These html lines not needed here for markdown.  But you could use something like this for the final website (which will be in HTML).